src/protocols/types/ContentType.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "content-type-v1.hpp"
+#include <cstdint>
+
+namespace NContentType {
+    wpContentTypeV1Type fromString(const std::string name);
```

Comment:
I wouldn't use `wpContentTypeV1Type` for internal handling, it's generated... Better to have our own internal one and `to`/`from` fns